qemu/tests/migration_multi_host_with_file_transfer.py

```diff
@@ -201,7 +201,7 @@ def migration_scenario(self):
 
             if (self.hostid == self.master_id()):
                 try:
-                    utils.run("dd if=/dev/urandom of=%s bs=1M"
+                    utils.run("dd if=/dev/zero of=%s bs=1M"
                               " count=%s" % (host_path, file_size))
```

Comment:
suggest to update the description of 
                    error.context("ping-pong between host and guest while"
                                  " migrating", logging.info)